src/bust.py

```diff
@@ -26,39 +26,61 @@
 
 
 class BustController:
-    def __init__(self, client: Client):
+    def __init__(
+        self,
+        client: Client,
+        current_channel_content: List[Tuple[Message, Attachment, str]],
+        current_channel: TextChannel,
+    ):
 
         # The actively connected voice client
-        self.active_voice_client: Optional[VoiceClient] = None
-        # The nickname of the bot. We need to store it as it will be
-        # changed while songs are being played.
-        self.original_bot_nickname: str = None
+        self.voice_client: Optional[VoiceClient] = None
         # Currently pinned "now playing" message ID
         self.now_playing_msg: Optional[Message] = None
         # Keep track of the coroutine used to play the next track, which is active while
         # waiting in the intermission between songs. This task should be interrupted if
         # stopping playback is requested.
         self.play_next_task: Optional[asyncio.Task] = None
 
+        # The nickname of the bot. We need to store it as it will be
+        # changed while songs are being played.
+        self.original_bot_nickname: str = ""
```

Comment:
`None` is actually [a valid state](https://docs.nextcord.dev/en/stable/api.html?highlight=edit#nextcord.Member.nick) for a guild member's nick - it just means use the username (i.e. no nick). A bot's nick may be `None` pre-bust.

So I think the type of original_bot_nickname` should remain an `Optional[str]`.